doc/visualizations.rst

```diff
@@ -8,37 +8,83 @@ Scikit-learn defines a simple API for creating visualizations for machine
 learning. The key feature of this API is to allow for quick plotting and
 visual adjustments without recalculation. We provide `Display` classes that
 expose two methods for creating plots: `from_estimator` and
-`from_predictions`. The `from_estimator` method will take a fitted estimator
-and some data (`X` and `y`) and create a `Display` object. Sometimes, we would
-like to only compute the predictions once and one should use `from_predictions`
-instead. In the following example, we plot a ROC curve for a fitted support
-vector machine:
+`from_predictions`.
+
+The `from_estimator` method generates a `Display` object from a fitted estimator and
+input data (`X`, `y`), and a plot.
+The `from_predictions` method creates a `Display` object from true and predicted
+values (`y_test`, `y_pred`), and a plot.
+
+For :term:`predict_proba`, the column corresponding to the probability estimate of
```

Comment:
I think we should mention that `from_predictions` prevents re-computing predictions. We had mentioned this twice previously but now we don't mention it once. I think the previous sentence here was fine, just minor tweaking:

"Using `from_predictions` avoids having to recompute predictions, but the user
needs to take care that the prediction values passed corresponding to the `pos_label`."